sdk/schemaregistry/azure-schemaregistry/CHANGELOG.md

```diff
@@ -13,9 +13,16 @@
 - `SchemaProperties` has the following instance variables: `id`, `format`, `version`.
 - `Schema` has the following properties: `properties` and `schema_definition`.
 - `SchemaFormat` provides the schema format to be stored by the service. Currently, the only supported format is `Avro`.
+- `api_version` has been added as a keyword arg to the sync and async `SchemaRegistryClient` constructors:
+  - currently defaults to and only accepts the string `"2021-10"`
```

Comment:
I probably would not highlight it in the changelog, docstring should be enough..

```suggestion
- `api_version` has been added as a keyword arg to the sync and async `SchemaRegistryClient` constructors.
```